File: Common/InMotionGIT.Common.Domain/Attributes/EntityEnglishDesigner.cs

namespace InMotionGIT.Common.Domain.Attributes;

/// <summary>
/// Define el titulo o nombre completo en ingles asociado a la clase.
/// </summary>
/// <remarks></remarks>
public sealed class EntityEnglishDesigner : EntityDesignerAttribute
{

    public EntityEnglishDesigner(string Title) : base((int)Enumerations.EnumLanguage.English, Title)
    {
    }

}

File: Common/InMotionGIT.Common - Copy/MetadataCatalogAttributes/EntityEnglishDesigner.vb
Namespace Attributes

    ''' <summary>
    ''' Define el titulo o nombre completo en ingles asociado a la clase.
    ''' </summary>
    ''' <remarks></remarks>
    Public NotInheritable Class EntityEnglishDesigner
        Inherits EntityDesignerAttribute

        Public Sub New(ByVal Title As String)
            MyBase.New(Enumerations.EnumLanguage.English, Title)
        End Sub

    End Class

End Namespace